distributed/tests/test_client.py

```diff
@@ -5288,6 +5288,19 @@ def test_client_timeout_2():
         assert stop - start < 1
 
 
+@gen_test()
+def test_client_active_bad_port():
+    import tornado.web
+    application = tornado.web.Application([
+        (r"/", tornado.web.RequestHandler),
+    ])
+    application.listen(8000)
+    with dask.config.set({'distributed.comm.timeouts.connect': '10ms'}):
+        c = Client('127.0.0.1:8000', asynchronous=True)
+        with pytest.raises((TimeoutError, IOError)):
+            yield c
```

Comment:
The other tests are failing because there is a lingering client.  If you want these tests to pass you can either close this client (`yield c.close(fast=True)`) or you can wait for https://github.com/dask/distributed/pull/2012 , which should solve this problem generally.